webpack.config.js

```diff
@@ -10,7 +10,8 @@ const webpackConfig = {
 	// The key is used as the name of the script.
 	entry: {
 		admin: './_inc/client/admin.js',
-		'static': './_inc/client/static.jsx'
+		'static': './_inc/client/static.jsx',
+		'modules-markdown-block': './modules/markdown/assets/js/jetpack-markdown-block.jsx',
```

Comment:
Why do we need to add an entry point here? Can we get away without one?

Reply:
Allow me to investigate this further, please. I saw this was the pattern used in other PRs for Gutenberg blocks in Jetpack and I just thought it was a valid solution.